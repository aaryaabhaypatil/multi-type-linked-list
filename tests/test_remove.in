REMOVE 0
REMOVE 1
REMOVE 2
NEW 1
A
REMOVE 0
