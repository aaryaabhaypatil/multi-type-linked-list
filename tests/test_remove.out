INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE
List 0: A
List 0 has been removed.

Number of lists: 0
